deepmd/pt/model/model/dp_model.py

```diff
@@ -1,83 +1,11 @@
 # SPDX-License-Identifier: LGPL-3.0-or-later
-from typing import (
-    Dict,
-    Optional,
-)
-
-import torch
-
-from deepmd.pt.model.atomic_model import (
-    DPAtomicModel,
-)
 from deepmd.pt.model.descriptor.base_descriptor import (
     BaseDescriptor,
 )
-from deepmd.pt.model.model.model import (
-    BaseModel,
-)
-from deepmd.pt.model.task.dipole import (
-    DipoleFittingNet,
-)
-from deepmd.pt.model.task.dos import (
-    DOSFittingNet,
-)
-from deepmd.pt.model.task.ener import (
-    EnergyFittingNet,
-    EnergyFittingNetDirect,
-)
-from deepmd.pt.model.task.polarizability import (
-    PolarFittingNet,
-)
 
-from .make_model import (
-    make_model,
-)
 
-
-@BaseModel.register("standard")
-class DPModel(make_model(DPAtomicModel)):
-    def __new__(
-        cls,
-        descriptor=None,
-        fitting=None,
-        *args,
-        # disallow positional atomic_model_
-        atomic_model_: Optional[DPAtomicModel] = None,
-        **kwargs,
-    ):
-        from deepmd.pt.model.model.dipole_model import (
-            DipoleModel,
-        )
-        from deepmd.pt.model.model.dos_model import (
-            DOSModel,
-        )
-        from deepmd.pt.model.model.ener_model import (
-            EnergyModel,
-        )
-        from deepmd.pt.model.model.polar_model import (
-            PolarModel,
-        )
-
-        if atomic_model_ is not None:
-            fitting = atomic_model_.fitting_net
-        else:
-            assert fitting is not None, "fitting network is not provided"
-
-        # according to the fitting network to decide the type of the model
-        if cls is DPModel:
-            # map fitting to model
-            if isinstance(fitting, EnergyFittingNet) or isinstance(
-                fitting, EnergyFittingNetDirect
-            ):
-                cls = EnergyModel
-            elif isinstance(fitting, DipoleFittingNet):
-                cls = DipoleModel
-            elif isinstance(fitting, PolarFittingNet):
-                cls = PolarModel
-            elif isinstance(fitting, DOSFittingNet):
-                cls = DOSModel
-            # else: unknown fitting type, fall back to DPModel
-        return super().__new__(cls)
+class DPModel:
```

Comment:
the name `DPModel` is a bit misleading, because the class it self is not a model